### ISSUE TITLE
Performance regression in Rocket League on 3.5

### ISSUE THREAD

**Issue by lostgoat** [ValveSoftware/SteamOS#1237]
* Game: Rocket League
* Performance issue: When vsync is disabled and "Disable "Frame Limit" is selected performance drops from 100fps to 60fps with heavy stutters.
* Notes: opening/closing the quick access menu "fixes" the performance issu.

Confirmed on 3.5.5. When in-game V-Sync is disabled and "Disable Frame Limit" is selected, the issue happens on several games I've tried. Oddly enough, when I open up the steam overlay, the stuttering stops completely. Rocket league was the most stark: ~110 fps w/ overlay open; and roughly 10 fps after closing it (note: it reports ~60 fps but there's absolutely no way that's the case)

_Originally posted by @slash-proc in https://github.com/ValveSoftware/SteamOS/issues/1176#issuecomment-1826820421_

**Comment by slash-proc** [comment]
While trying to reproduce #1176, I noticed the exact same behavior I mentioned in my original comment in both Rocket League and CarX Drift Racing Online.

After updating to 3.5.7 the problem is now gone. I'm not sure if I incorrectly associated the problem with it but either way, the problem I noticed now gone in my testing 

Given this change, I'd say this issue can be closed. I appreciate the vigilance.
